package.json

```diff
@@ -31,6 +31,7 @@
     "@RoleModel:registry": "https://npm.pkg.github.com"
   },
   "dependencies": {
+    "@rolemodel/rolemodel-design-system": "^0.0.6-beta",
```

Comment:
Why are we adding ourselves as a dependency?

Reply:
Okay, removed.